uat/testing-features/src/main/resources/greengrass/features/shadow-8.feature

```diff
@@ -0,0 +1,219 @@
+@Shadow @Shadow-8 @Greengate @beta
+Feature: Shadow-8
+
+  As a developer, I can set the sync directionality.
+
+  Background:
+    Given my device is registered as a Thing
+    And my device is running Greengrass
+    And I start an assertion server
+    When I add random shadow for MyThing with name MyThingNamedShadow in context
+    When I add random shadow for MyThing2 with name MyThingNamedShadow2 in context
+
+  @functional @JavaSDK @smoke @RunWithHSM
+  Scenario: Shadow-8-T1: As a developer, I can sync a local named shadow between cloud and device.
+    When I create a Greengrass deployment with components
+      | aws.greengrass.Nucleus       | LATEST  |
+      | aws.greengrass.ShadowManager | LATEST |
+    And I update my Greengrass deployment configuration, setting the component aws.greengrass.ShadowManager configuration to:
+        """
+        {
+           "MERGE":{
+              "synchronize":{
+                 "direction": "betweenDeviceAndCloud",
+                 "shadowDocuments":[
+                    {
+                       "thingName":"${MyThing}",
+                       "classic":false,
+                       "namedShadows":[
+                          "${MyThingNamedShadow}"
+                       ]
+                    },
+                    {
+                       "thingName":"${MyThing2}",
+                       "classic":true
+                    }
+                 ],
+                 "provideSyncStatus":true,
+                 "maxOutboundSyncUpdatesPerSecond":50
+              },
+              "shadowDocumentSizeLimitBytes":8192,
+              "maxDiskUtilizationMegaBytes":16
+           }
+        }
+        """
+    And I update my Greengrass deployment configuration, setting the component aws.greengrass.Nucleus configuration to:
+        """
+        {"MERGE":{"logging": {"level": "DEBUG"}}}
+        """
+    And I deploy the Greengrass deployment configuration
+    Then the Greengrass deployment is COMPLETED on the device after 4 minutes
+    When I install the component ShadowComponentPing from local store with configuration
+        """
+        {
+           "MERGE":{
+                "assertionServerPort": ${assertionServerPort},
+                "Operation": "UpdateThingShadow",
+                "ThingName": "MyThing",
+                "ShadowName": "MyThingNamedShadow",
+                "ShadowDocument": "{\"state\":{\"reported\":{\"color\":{\"r\":255,\"g\":255,\"b\":255},\"SomeKey\":\"SomeValue\"}}}",
+                "ExpectedShadowDocument": "{\"version\":1,\"state\":{\"reported\":{\"color\":{\"r\":255,\"g\":255,\"b\":255},\"SomeKey\":\"SomeValue\"}}}"
```

Comment:
All of the shadow documents should have `\\\`. Other than that, rest of the PR looks good.